src/Core/Models/Domain/Login.cs

```diff
@@ -44,7 +44,11 @@ public async Task<LoginView> DecryptAsync(string orgId, SymmetricCryptoKey key =
                 view.Uris = new List<LoginUriView>();
                 foreach (var uri in Uris)
                 {
-                    view.Uris.Add(await uri.DecryptAsync(orgId, key));
+                    var loginUriView = await uri.DecryptAsync(orgId, key);
+                    if (bypassValidation || (await uri.ValidateChecksum(loginUriView.Uri, orgId, key)))
```

Comment:
⛏️ No need for the extra parenthesis:
```suggestion
                    if (bypassValidation || await uri.ValidateChecksum(loginUriView.Uri, orgId, key))
```